src/server/api/router.js

```diff
@@ -30,6 +30,9 @@ export default function apiRouter() {
         terrainHandler({
             method: "GET",
             pathname: "/dashboard",
+            headers: {
+                "Content-Type": "application/json",
+            },
```

Comment:
Is this actually needed for a GET request?

Reply:
@johnworth had it in the fetch-request. So I copied it here.

Reply:
I think our Swaggerized terrain endpoints use this header to decide what format to use in the response. So this content-type tells terrain we want a JSON response, but requesting `application/x-yaml` content-type will return a YAML response.